### ISSUE TITLE
Make "Support Site" more visible on add-on page 

### ISSUE THREAD

**Issue by erosman** [mozilla/addons-frontend#4432]
moved from: https://github.com/mozilla/addons-server/issues/7483

For years, I have been asked "Where is the support site?" even though they have ALWAYS been listed under "Support Site" on add-on's pages. Users then use the Review system for support queries.

The new layout made it even less noticeable. Please make it more dominant and noticeable.

**Comment by tofumatt** [comment]
cc'ing @pwalm: this is a fair point; for a lot of add-ons the support site can be buried but it's a really good point of contact for users to get help with their add-on. Would it be worth highlighting its visibility elsewhere on the detail page? Maybe it's not worth it, that's fine too, but wondering what UX thinks about this.

**Comment by pwalm** [comment]
Yes! I'd like to emphasize the support site in two ways: bigger on the page (obvs), and through the eventual changes to the ratings and review flow (surfacing the support site through triage).

**Comment by erosman** [comment]
Please also consider adding an ICON to improve visibility.

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDFRNT-130
